src/ray/raylet/worker_pool.cc

```diff
@@ -130,20 +130,20 @@ WorkerPool::WorkerPool(instrumented_io_context &io_service, const NodeID node_id
 }
 
 WorkerPool::~WorkerPool() {
-  std::unordered_set<Process> procs_to_kill;
```

Comment:
I'm wondering if a plain vector would work.

Reply:
I think the answer is yes. Given that this is not performance critical, I would think keeping a hash map is slightly better preferable since it makes is easier to switch StartupToken to a random ID if we so desire. But very open to be convinced on this, let me know what you think :)

If this was in an inner loop I would most definitely prefer a vector over a hash map!

Reply:
Oh I was not worrying about performance. I was just wondering if there were duplicated process so we used a set.